redbadger/continous-compliance
# Issues

Closes #42 

## How to test

1. Go to `count-dracula` code and pull recent remote branches
```sh
git fetch
```
2. Go to branch `test-continous-compliance-v09`
```sh
git checkout test-continous-compliance-v09
```
3. Do an empty commit and push to remote
```sh
git commit --allow-empty -m "Trigger continous-compliance CI/CD"
git push
```
4. Go to count dracula [actions](https://github.com/redbadger/count-dracula/actions)
5. Wait until `continous compliance` step on **continous-compliance** job finished and open the job.
6. Click the URL on the last line and should trigger a file download
7. open `github` sub folder and check of a file `info.json`
8. You should see on json a new key called `issues` with the issues information
9. 🚀 
![](https://media.giphy.com/media/4Mni3cxTuKHDi/giphy.gif)
